packages/venia-ui/lib/drivers/adapter.js

```diff
@@ -116,8 +116,20 @@ const VeniaAdapter = props => {
 // Create a new store link to include store codes and currency in the request
 VeniaAdapter.storeLink = setContext((_, { headers }) => {
     const storeCurrency = storage.getItem('store_view_currency') || null;
-    const storeCode = storage.getItem('store_view_code') || STORE_VIEW_CODE;
+    // Getting store code from domains.json config
+    const storeCodeFromDomainsConfig = STORE_FROM_DOMAINS;
```

Comment:
This essentially publicizes ownership information for multiple websites. Is this something normally kept private/secret?